articles/configure-your-site-navigation.mdx

```diff
@@ -2,103 +2,106 @@
 
-## Configure dashboard actions
+At the bottom of the scrollable section in your *Site Navigation* are your Developer tools. These are important for managing the Site itself, rather than it's content.
 
-The CloudCannon dashboard has an actions section for quickly navigating to specific pages.
+* **Status** — This page has suggestions for Site configuration, summarizes your Site's status, and provides logs for your *Syncs*, *Builds*, and *Build Deploys*.
+* **Files** — Open the File browser to see all the source files for your Site in the same file structure as your Git repository.
+* **Site Settings** — All your CloudCannon settings for this Site, including settings for your files, builds, hosting, sharing, and more.
 
-### Update Home
-
-The *Update Home* button will open your home page (or any page of your choice) in the [Visual Editor](/documentation/articles/introducing-the-visual-editor). This is configured in your [global configuration file](/documentation/articles/create-a-global-configuration-file/) using the key `default_path` within `editor`.
-
-To configure which path this button will open, use the relative URL of the page.
-
-<comp.MultiCodeBlock language="YAML" translate_into={["json"]} source="cloudcannon.config.yml">
-```
-editor:
-  default_path: /about/
-```
-</comp.MultiCodeBlock>
+<comp.Notice info_type="info">
+  The Developer section is only available for team members with specific [permissions](/documentation/articles/what-are-permission-groups/).
+</comp.Notice>
 
-Clicking the **Update Home** dashboard action loads the `default_path`. The example above loads `/about/` rather than the default `/`.
\ No newline at end of file
+## Options
+
+Your Site Navigation has the following options available:
+
+<comp.DataReference>
+  <comp.DataReferenceRow label="collection_groups" type_markdown="Array">
+    This key defines the groups and order of collections in your *Site Navigation*. Each item in the array must contain the `heading` and `collections` keys.
+
+    <comp.MultiCodeBlock language="YAML" translate_into={["json"]} source="cloudcannon.config.yml">
+                                                            ``````````
+
+                                                            collection_groups:
+                                                              - heading___1___: Content
+                                                                collections___2___:
+                                                                  - pages
+                                                                  - blog
+                                                              - heading: Data Files
+                                                                collections:
+                                                                  - data
+
+                                                            ``````````
```

Comment:
I need to fix this snippet that likes indenting things.

````````````````````suggestion

    <comp.MultiCodeBlock language="YAML" translate_into={["json"]} source="cloudcannon.config.yml">
    ``````````
    collection_groups:
      - heading___1___: Content
        collections___2___:
         - pages
         - blog
      - heading: Data Files
        collections:
         - data
    ``````````
````````````````````